gallery/base.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>SPEC - Gallary</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:300i,400,700" rel="stylesheet">
+  <link rel="stylesheet" href="fonts/icomoon/style.css">
+
+  <link rel="stylesheet" href="css/bootstrap.min.css">
+  <link rel="stylesheet" href="css/magnific-popup.css">
+  <link rel="stylesheet" href="css/jquery-ui.css">
+  <link rel="stylesheet" href="css/owl.carousel.min.css">
+  <link rel="stylesheet" href="css/owl.theme.default.min.css">
+
+  <link rel="stylesheet" href="css/lightgallery.min.css">
+
+  <link rel="stylesheet" href="css/bootstrap-datepicker.css">
+
+  <link rel="stylesheet" href="fonts/flaticon/font/flaticon.css">
+
+  <link rel="stylesheet" href="css/swiper.css">
+
+  <link rel="stylesheet" href="css/aos.css">
+
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/custom.css">
+
+</head>
+
+<body>
+
+  <div class="site-wrap">
+
+    <div class="site-mobile-menu">
+      <div class="site-mobile-menu-header">
+        <div class="site-mobile-menu-close mt-3">
+          <span class="icon-close2 js-menu-toggle"></span>
+        </div>
+      </div>
+      <div class="site-mobile-menu-body"></div>
+    </div>
+
+
+
+
+    <header class="site-navbar py-3" role="banner">
+
+  <div class="container-fluid">
+    <div class="row align-items-center">
+
+      <div class="col-6 col-xl-2" data-aos="fade-down">
+        <!-- <h1 class="mb-0"><a href="index.html" class="text-black h2 mb-0">SPEC</a></h1>-->
+        <a href="https://specnith.com"><img src="bg-img/spec_logo.png" class="logo" /></a>
+      </div>
+      <div class="col-10 col-md-8 d-none d-xl-block" data-aos="fade-down">
+        <nav class="site-navigation position-relative text-right text-lg-center" role="navigation">
+
+          <ul class="site-menu js-clone-nav mx-auto d-none d-lg-block">
+            <li><a href="https://specnith.com/index.html#about_index">About</a></li>
+            <li><a href="https://specnith.com/workshops.html">Workshops</a></li>
+            <li><a href="https://specnith.com/blog.html">Blogs</a></li>
+            <li><a href="https://specnith.com/team_page_final.html">Team SPEC</a></li>
```

Comment:
@sjainit @baamaboi don't you think this should be local. I guess this will take time for our site to load.

Reply:
They are only meant as a demo and will be changed